### DISCUSSION TITLE
Compiling Phazerville 1.8 for Teensy 4.0 failed due to EEPROM error

### DISCUSSION THREAD

**Discussion by Peter-ui-dotcom** [djphazer/O_C-Phazerville#102] [Q&A]
I'm using Arduino IDE 232. When compiling Phazerville 1.8. for Teensy 4.0 it fails with the message: 
Compilation error: static assertion failed: EEPROM Allocation Exceeded .
For Teensy 4.1. it works, but unfotunately I have only 4.0 in my O_C.
Any Idea? Thanks

**Comment by djphazer** [comment]
I have a hack in place to maximize the emulated EEPROM space on T4.0 here - https://github.com/djphazer/O_C-Phazerville/blob/phazerville/software%2Fsrc%2Fextern%2Favr%2Feeprom.h

This is meant to override a header in the standard Teensy library; it's possible the Arduino IDE is finding the standard one instead.

Otherwise, you might not be able to fit all the apps... worst case scenario, leave several of the app flags commented out in `OC_options.h`

**Reply by Peter-ui-dotcom** [reply]
Thanks, that worked. I put your hack at the begin in ~$USER\AppData\Local\Arduino15\packages\teensy\hardware\avr\1.59.0\libraries\EEPROM\EEPROM.h.  But nevertheless I had to remove a few Applications. I'm currently finetuning it
